docs/installing-solidity.rst

```diff
@@ -6,6 +6,28 @@
 Installing Solidity
 ###################
 
+Versioning
```

Comment:
My reasoning was to explain the version before they select it for using Solidity. Maybe the versioning should have two sections, a short intro, and a detailed description.